StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentMethodType.swift

```diff
@@ -164,29 +164,35 @@ extension PaymentSheet {
                 recommendedStripePaymentMethodTypes.map { PaymentMethodType.stripe($0) }
                 // External Payment Methods
                 + elementsSession.externalPaymentMethods.map { PaymentMethodType.external($0) }
+            
+            let hasIneligibleConfiguration = configuration.billingDetailsCollectionConfiguration.email == .never && configuration.defaultBillingDetails.email?.isEmpty != false
```

Comment:
Nit: Since this is only ineligible for the Bank tab, you can move it into `eligibleForInstantDebits` and `eligibleForLinkCardBrand`, and maybe make it an extension property on `BillingDetailsCollectionConfiguration`.

Reply:
Good suggestion! Done in: https://github.com/stripe/stripe-ios/pull/4162/commits/06a564047490706b05c691ab21c1a896bbb25ba2